app/models/micropost.rb

```diff
@@ -0,0 +1,29 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
```

Comment:
thêm association belongs_to user cho micropost, 1 post thuộc về 1 user. Tương ứng ở model User phải thêm association has_many để mô tả quan hệ one-to-many giữa user và post